### ISSUE TITLE
Timesheet UX Issue: "Resume Timer" -> "Show Timer"

### ISSUE THREAD

**Issue by aleksas** [frappe/erpnext#16587]
I may be wrong but it looks that the caption on the button should be **Show Timer** not **Resume Timer** because timer was not paused to be resumed, button just shows the timer popup window.

![image](https://user-images.githubusercontent.com/594470/52329643-01c9a380-29fc-11e9-9661-265c5e9329c4.png)

**[LINE THAT PROBABLY NEEDS TO BE CHANGED](https://github.com/frappe/erpnext/blob/d254af9fea74ea06591687de0e7f82eb7f4ff85b/erpnext/projects/doctype/timesheet/timesheet.js#L59)**

@rmehta, @surajshetty

**Comment by rasos** [comment]
Agree, that should be named "Show Timer". Wondering if it is paused with job cards https://erpnext.com/blog/releases/erpnext-v-12-7-0
